extensions/functions_datetime.yaml

```diff
@@ -218,6 +218,58 @@ scalar_functions:
           - name: y
             value: interval_day
         return: timestamp
+  -
+    name: "multiply_interval"
```

Comment:
Should this be just another multiply specialization as add is above?

Reply:
+1.  It seems a little odd to have `add`, `subtract`, and then `multiply_interval` instead of `multiply`.

Reply:
Ok, operator renamed.